docs/blog/newsletter-may-2019.rst

```diff
@@ -0,0 +1,130 @@
+
+Finally, just a reminder: If you spot something on Slack and you think, "That was interesting/useful/cool, we should save that", please go ahead and pop the ``:suggest-for-newsletter:`` emoji on it. That'll bring it to our attention, so we can write it up into a newsletter article and save for posterity!
+
+Speaking of which: what interesting things have come up this month?
+
+--------------------------------------
+Collecting and acting on user feedback
+--------------------------------------
+
+This month, a Slack discussion produced a bonanza of insights into how documentarians collect and apply user feedback. Here's a roundup:
+
+* Use Google Analytics to collect traffic information, so you can identify the best way to collect feedback. For example, if you have low traffic but high engagement, you might get great feedback from contacting users by email. See the `Docs by Design <https://docsbydesign.com/>`_ blog for more about tailoring your approach.
+* Collect feedback with forms at the end of each page, or provide a feedback email address, chat widget, or survey link. Then sort and group into actionable doc improvements. Some suggested asking targeted questions or having users rank the page instead of requesting open-ended comments. 
+* Use services like `MongoDB Stitch <https://www.mongodb.com/cloud/stitch>`_ to push docs feedback directly to a Slack channel.
+* Allow users to directly log Salesforce cases/GitHub issues/Jira tickets.
+* Contact users one-on-one to ask for feedback or arrange to receive regular ongoing feedback. Don't forget to close the loop: let these users know when you improve something based on their feedback.
+
+The two biggest challenges were prioritizing feedback-based improvements without compromising ongoing documentation efforts, and finding the time to make use of the collected feedback. The main strategy people used for addressing these challenges was holding document improvement sessions to discuss and prioritize the backlog.
+
+And of course, some WTD talks have covered user feedback too:
+
+* `CSAT - what’s that? </videos/na/2016/csat-what-s-that-betsy-roseberg/>`_
+* `Feedback handling, community wrangling, panhandling </videos/eu/2016/feedback-handling-community-wrangling-panhandling-chris-mills/>`_
+* `You have already succeeded 2.0 </videos/eu/2017/you-have-already-succeeded-design-critique-guidelines-make-feedback-easier-2-0-christy-lutz/>`_
+
+------------------------------------
+Tips and tricks for producing videos
+------------------------------------
+
+Videos can be a great way of educating users, but it's not always easy to do them well, especially if you haven't worked on video before. Here are some tips from a recent community community:
```

Comment:
"community" is repeated at the end of this intro sentence

Reply:
fixed!